domain/src/main/java/com/thoughtworks/go/domain/materials/git/GitCommand.java

```diff
@@ -93,12 +93,12 @@ private String modificationTemplate(String separator) {
     }
 
     public List<Modification> latestModification() {
-        return gitLog("-1", "--date=iso", "--pretty=medium", remoteBranch());
+        return gitLog("-1", "--date=iso", "--pretty=medium", "--no-decorate", "--no-color", remoteBranch());
 
     }
 
     public List<Modification> modificationsSince(Revision revision) {
-        return gitLog("--date=iso", "--pretty=medium", String.format("%s..%s", revision.getRevision(), remoteBranch()));
+        return gitLog("--date=iso", "--pretty=medium", "--no-decorate", "--no-color", String.format("%s..%s", revision.getRevision(), remoteBranch()));
```

Comment:
Why not add these extra args in the method `gitLog`, since they hold true for all log commands?

Reply:
By the same token, `--date=iso` and `--pretty=medium` should be moved into `gitLog`

I decided it was more consistent to add the arguments there

@varshavaradarajan